public/assets/jquery-migrate/jquery-migrate-3.0.0.min.js

```diff
@@ -0,0 +1,2 @@
+/*! jQuery Migrate v3.0.0 | (c) jQuery Foundation and other contributors | jquery.org/license */
```

Comment:
TODO
Plan to remove this after assets are converted to use the standard Header class

Reply:
Brady, if not mistaken the JQuery migrate should show any conflicts in console and i'm pretty sure author used it for that reason but just wanted to remind while testing and before removal.

Reply:
Places that this was used is testing well on openemr standard jquery version, so will remove this.